ios-base.xcodeproj/project.pbxproj

```diff
@@ -1048,12 +1054,13 @@
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				INFOPLIST_PREPROCESS = NO;
-				IPHONEOS_DEPLOYMENT_TARGET = 9.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.2;
```

Comment:
Please do not change this. We will update or minor deployment target but it is still in discussion and it should be in a different PR.

Reply:
agreed, I had changed it only for testing with available simulators and should have reverted, fixed now